packages/app/src/components/VisualizationOptions/Options/ColorSet.js

```diff
@@ -47,8 +47,8 @@ const ColorSet = ({ value, onChange, disabled }) => (
                         label: i18n.t('Color blind'),
                     },
                     {
-                        id: COLOR_SET_MONO_PATTERNS,
-                        label: i18n.t('Mono patterns'),
+                        id: COLOR_SET_PATTERNS,
```

Comment:
This requires an Analytics change, which isn't listed in the description. Please add it and only merge this PR after the corresponding Analytics change has been merged and the dep version has been bumped in this PR.